k8s-manifests/database/deployment.yml

```
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+
+metadata:
+ name: mongo-dep
+ namespace: three-tier
+ labels:
+   app: mongodb
+
+spec:
+  replicas: 2
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 1 but found 2

(indentation)